Extensions/pokes.js

```diff
@@ -94,7 +94,7 @@ XKit.extensions.pokes = {
 	fetchPoke: function(db_nr, pokedThing) {
 		GM_xmlhttpRequest({
 			method: "GET",
-			url: "https://gist.githubusercontent.com/ThePsionic/54a1f629dba66e53aaa4/raw/73da44a7295f99c3cd7e29cd8012bbbe341a78d8/pokedex.json",
+			url: "https://gist.githubusercontent.com/ThePsionic/54a1f629dba66e53aaa4/raw/6dddd7504ff466e466d4f3b0b3c9574bdb221d7d/pokedex.json",
```

Comment:
if you change that line to `https://gist.githubusercontent.com/ThePsionic/54a1f629dba66e53aaa4/raw/pokedex.json` It should automatically always use the latest gist I think